examples/session_view.cpp

```diff
@@ -70,21 +70,21 @@ void session_view::render()
 
 	float seconds = (m_timestamp[0] - m_timestamp[1]) / 1000000.f;
 
-	int download_rate = (m_cnt[0][m_recv_payload_idx] - m_cnt[1][m_recv_payload_idx])
-		/ seconds;
-	int upload_rate = (m_cnt[0][m_sent_payload_idx] - m_cnt[1][m_sent_payload_idx])
-		/ seconds;
+	int download_rate = int((m_cnt[0][m_recv_payload_idx] - m_cnt[1][m_recv_payload_idx])
+		/ seconds);
+	int upload_rate = int((m_cnt[0][m_sent_payload_idx] - m_cnt[1][m_sent_payload_idx])
+		/ seconds);
 
 	pos += snprintf(str, sizeof(str), "%s%s fail: %s down: %s (%s) "
 		"  bw queue: %s | %s conns: %3d  unchoked: %2d / %2d "
 		"                                      %s\x1b[K"
 		, esc("48;5;238")
 		, esc("1")
-		, add_suffix(m_cnt[0][m_failed_bytes_idx]).c_str()
-		, color(add_suffix(download_rate, "/s"), col_green).c_str()
-		, color(add_suffix(m_cnt[0][m_recv_payload_idx]), col_green).c_str()
-		, color(to_string(m_cnt[0][m_limiter_up_queue_idx], 3), col_red).c_str()
-		, color(to_string(m_cnt[0][m_limiter_down_queue_idx], 3), col_green).c_str()
+		, add_suffix(float(m_cnt[0][m_failed_bytes_idx])).c_str()
+		, color(add_suffix(float(download_rate), "/s"), col_green).c_str()
```

Comment:
I wonder if it would make more sense to add an overload to add_suffix that takes an int, or int64_t